### PR TITLE
Inconsistencies with buttons and shadows + UI of detailEventScreen

### PR DESCRIPTION
**Pull request by leagrieder**
This pull request aims to enhance the consistency of the buttons across the app. Specifically, we have standardized the use of shadows to indicate interactive elements. Shadows are now exclusively applied to buttons and other clickable components, ensuring that static components are visually distinguished.

To achieve this, I have wrapped all clickable icons in a box with a shadow. This change clearly indicates their interactivity, improving the overall user experience and making the app more intuitive.

Moreover, following the logic of making buttons consistent throughout the app, the DetailedEventScreen has been modified as well (see Figma) to incorporate this logic. I had to modify the whole UI of this screen because we had buttons like "report a problem" that we thought would be included for M4 but will not be. 

Note that the beds and the parking spots will be visible on this screen in another pull request to not make this one to big

View for Owner:
image

View for guest: 
image

### CODE REVIEW

File: app/src/androidTest/java/com/monkeyteam/chimpagne/newtests/ui/ViewDetailEventScreenTests.kt [hunk 1]
```diff
@@ -214,10 +283,13 @@ class ViewDetailEventScreenTests {
...
   fun testShareButton() {
-    val event = TEST_EVENTS[0]
+    val event = TEST_EVENTS[2]
 
     val eventVM = EventViewModel(event.id, database)
 
+    accountViewModel.loginToChimpagneAccount(TEST_ACCOUNTS[1].firebaseAuthUID, {}, {})
+    accountManager.signInTo(TEST_ACCOUNTS[1])
+
     while (eventVM.uiState.value.loading) {}
```
Comment (arnaudlvq) [inline app/src/androidTest/java/com/monkeyteam/chimpagne/newtests/ui/ViewDetailEventScreenTests.kt:293]: You correctly signin as a User. Great

File: app/src/main/java/com/monkeyteam/chimpagne/ui/FindEventScreen.kt [hunk 2]
```diff
@@ -330,24 +331,27 @@ fun FindEventFormScreen(
...
-                Spacer(Modifier.width(8.dp))
-                Text(
-                    stringResource(id = R.string.find_event_search_button_text).uppercase(),
-                    style = ChimpagneTypography.bodyLarge)
+        Box(modifier = Modifier.padding(8.dp).fillMaxWidth()) {
+          Button(
+              onClick = { onSearchClick() },
+              modifier =
+                  Modifier.shadow(10.dp, RoundedCornerShape(12.dp))
+                      .clip(RoundedCornerShape(12.dp))
```
Comment (arnaudlvq) [inline app/src/main/java/com/monkeyteam/chimpagne/ui/FindEventScreen.kt:339]: Nice UI touch

File: app/src/main/java/com/monkeyteam/chimpagne/ui/FindEventScreen.kt [hunk 3]
```diff
@@ -511,7 +515,7 @@ fun FindEventMapScreen(
               modifier =
                   Modifier.padding(start = 12.dp, top = 12.dp)
                       .testTag("go_back")
-                      .shadow(elevation = 4.dp, shape = RoundedCornerShape(100))
+                      .shadow(elevation = 10.dp, shape = RoundedCornerShape(12.dp))
```
Comment (arnaudlvq) [inline app/src/main/java/com/monkeyteam/chimpagne/ui/FindEventScreen.kt:518]: 10 is a lot for elevation. In the future, try to lowel the elevation

see the Official [Guidlines](https://designfornative.com/basics-of-elevation-on-android/l)

image

File: app/src/main/java/com/monkeyteam/chimpagne/ui/components/Button.kt [hunk 4]
```diff
@@ -95,8 +106,8 @@ fun IconTextButton(
       verticalAlignment = Alignment.CenterVertically,
       modifier =
           modifier
-              .shadow(elevation = 4.dp, shape = RoundedCornerShape(100))
-              .background(shape = RoundedCornerShape(100), color = color)
+              .shadow(elevation = 10.dp, shape = RoundedCornerShape(12.dp))
```
Comment (arnaudlvq) [inline app/src/main/java/com/monkeyteam/chimpagne/ui/components/Button.kt:109]: same here

File: app/src/main/java/com/monkeyteam/chimpagne/ui/components/eventview/EventMainInfo.kt [hunk 5]
```diff
@@ -118,23 +108,7 @@ fun EventMainInfo(event: ChimpagneEvent) {
...
-                          ContextCompat.getString(context, R.string.deep_link_url_event) + event.id)
-                    }
-                    clipboardManager.setText(annotatedString)
-                  },
-                  modifier = Modifier.size(36.dp).testTag("share")) {
-                    Icon(
-                        imageVector = Icons.Rounded.Share,
-                        contentDescription = "Share Event",
-                        tint = MaterialTheme.colorScheme.onPrimaryContainer)
+                        color = MaterialTheme.colorScheme.onPrimaryContainer)
```
Comment (arnaudlvq) [inline app/src/main/java/com/monkeyteam/chimpagne/ui/components/eventview/EventMainInfo.kt:111]: Nice little bug fix, tint and color are often confused with each other

File: app/src/main/java/com/monkeyteam/chimpagne/ui/components/eventview/OrganiserView.kt [hunk 6]
```diff
@@ -41,41 +59,57 @@ fun OrganiserView(ownerId: ChimpagneAccountUID, accountViewModel: AccountViewMod
...
+      horizontalArrangement = Arrangement.SpaceBetween,
+      modifier = Modifier.fillMaxWidth().padding(horizontal = 16.dp)) {
+        Row {
+          profilePictureUriState.value?.let { uri ->
+            ProfileIcon(
+                uri = uri,
+                onClick = {
+                  Toast.makeText(
+                          context,
+                          "This function will be implemented in a future version",
```
Comment (arnaudlvq) [inline app/src/main/java/com/monkeyteam/chimpagne/ui/components/eventview/OrganiserView.kt:73]: Should be removed

File: app/src/main/java/com/monkeyteam/chimpagne/ui/components/eventview/OrganiserView.kt [hunk 7]
```diff
@@ -41,41 +59,57 @@ fun OrganiserView(ownerId: ChimpagneAccountUID, accountViewModel: AccountViewMod
...
+                  "${
+                    stringResource(
+                        id = R.string.event_details_screen_organized_by
+                    )
+                }\n ${accountViewModel.uiState.value.fetchedAccounts[ownerId]?.firstName ?: ""} ${accountViewModel.uiState.value.fetchedAccounts[ownerId]?.lastName ?: ""}",
               fontSize = 14.sp,
               fontFamily = ChimpagneFontFamily,
               color = MaterialTheme.colorScheme.onPrimaryContainer,
-              modifier = Modifier.padding(start = 16.dp))
+              modifier = Modifier.padding(start = 20.dp, end = 20.dp))
```
Comment (arnaudlvq) [inline app/src/main/java/com/monkeyteam/chimpagne/ui/components/eventview/OrganiserView.kt:88]: seems like a lot of wasted space on small screen

**Review by arnaudlvq — APPROVED** [review]
Overall looks good, thanks for providing screenshots. I think your personal touch gives Chimpagne its signature in some way 😎

**Review by Athanaze — APPROVED** [review]
I agree with Arnaud, Toast of unimplemented features should be removed in the future

**Comment by Athanaze** [comment]
> PanelBars.kt

Regarding the conflict with panelBars, I removed the top bar that was in it, to use the same top bar component everywhere. The merge conflict should be easy to resolve.

This is ready to be merged to main in my opinion.

File: app/src/main/java/com/monkeyteam/chimpagne/ui/FindEventScreen.kt (hunk 3 above)
Comment (leagrieder) [reply app/src/main/java/com/monkeyteam/chimpagne/ui/FindEventScreen.kt]: I changed this as requested :) Thank you for the input!

**Comment by leagrieder** [comment]
@arnaudlvq and @Athanaze thank you for the reviews. I totally agree that "this function will be implemented in a future version" toasts should be removed (as discussed in our meetings). But I have chosen not to remove this one specifically because @juaniac is currently working on the polls so this Toast will be removed as soon as his PR is merged. I hope you agree with this.

**Pull request merged**